Clean unused variables, keep just TCL/TK_INCLUDE/LIB_SPEC

--- Init.make.in~	2012-01-25 13:43:05.000000000 +0100
+++ Init.make.in	2013-03-07 14:33:25.414055485 +0100
@@ -1,7 +1,7 @@
 # -*- Makefile -*-
 # @configure_input@
 #
-# Copyright (C) 1996-2004 LAAS/CNRS 
+# Copyright (C) 1996-2004,2013 LAAS/CNRS 
 # All rights reserved.
 # 
 # Redistribution and use in source and binary forms, with or without
@@ -51,7 +51,6 @@
 MKINSTALLDIRS=	$(SHELL) $(abs_srcdir)/mkinstalldirs
 
 #CPPFLAGS= 	-I$(GDHE_CLIENT_SRC) -I$(GDHE_TOGL_SRC)
-CFLAGS=		@TCL_EXTRA_CFLAGS@
 
 # Custom parameters for building the module
 
@@ -66,17 +65,10 @@
 LTLDPURE=	$(LIBTOOL) --mode=link purify $(CC)
 
 # Tcl/Tk
-TCL_CC=			@TCL_CC@
-TCL_CPPFLAGS=		@TCL_CPPFLAGS@
-TCL_LDFLAGS=		@TCL_LDFLAGS@
+TCL_CC=			${LTCC}
+TCL_CPPFLAGS=		@TCL_INCLUDE_SPEC@ @TK_INCLUDE_SPEC@
 TCL_LIBS=		@TCL_LIB_SPEC@ @TK_LIB_SPEC@
-TCL_LIBDIR=		@LIB_RUNTIME_DIR@
-TCL_LIB_FLAGS=		@TCL_LIB_FLAGS@
-TCL_DBGX=		@TCL_DBGX@
-TCL_SHLIB_CFLAGS=	@TCL_SHLIB_CFLAGS@
-TCL_SHLIB_LDFLAGS=	@TCL_SHLIB_LDFLAGS@
-TCL_SHLIB_SUFFIX=	@TCL_SHLIB_SUFFIX@
-TCL_SHLIB_LD=		@TCL_SHLIB_LD@
+TCL_SHLIB_LD=		${LIBTOOL} --tag=CC --mode=link ${CC} -module -rpath ${exec_prefix}/lib
 
 # Other libs
 LIBS=			@LIBS@
